specs/swagger.yml

```diff
@@ -5041,6 +5041,9 @@ definitions:
         type: string
       address:
         type: string
+      cost:
+        type: integer
```

Comment:
same as above for minimum and maximum values. Swagger defaults integer to 32bits.

```
   minimum: 0
   maximum: 65535
```

Reply:
might want to define cost in 1 place under definitions and reuse.

Reply:
Fixed